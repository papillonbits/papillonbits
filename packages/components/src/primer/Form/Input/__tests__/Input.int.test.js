import { regular, inactive } from './Input.int.story'

describe('<Input />', () => {
  afterEach(() => jest.clearAllMocks())

  describe('Render', () => {
    test('must match regular()', () => {
      expect(global.renderToJSON(regular())).toMatchSnapshot()
    })

    test('must match inactive()', () => {
      expect(global.renderToJSON(inactive())).toMatchSnapshot()
    })
  })
})
