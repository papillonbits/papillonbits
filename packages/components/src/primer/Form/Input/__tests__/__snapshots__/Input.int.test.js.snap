// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Input /> Render must match inactive() 1`] = `
<div>
  <input
    aria-label="Repository description"
    class="form-control input-inactive"
    placeholder="Fantastic placeholder"
    type="text"
    value=""
  />
</div>
`;

exports[`<Input /> Render must match regular() 1`] = `
<div>
  <input
    aria-label="Repository description"
    class="form-control"
    placeholder="Fantastic placeholder"
    type="text"
    value=""
  />
</div>
`;
